api/src/main/java/org/geysermc/geyser/api/GeyserApi.java

```diff
@@ -116,4 +117,12 @@ public interface GeyserApi extends GeyserApiBase {
     static GeyserApi api() {
         return Geyser.api(GeyserApi.class);
     }
+
+    /**
+     * Gets the {@link org.geysermc.api.util.ApiVersion} Geyser API version
+     * @return the current geyser api version
+     */
+     default org.geysermc.api.util.ApiVersion geyserApiVersion() {
+        return new ApiVersion(2, 1, 1);
+     }
```

Comment:
space between the description and the first tag. also, import ApiVersion

Reply:
fixed in https://github.com/GeyserMC/Geyser/pull/3880/commits/123ae7d9821ea167c23346d15f529f4d0916c95d, thanks